dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -189,3 +189,10 @@ def test_cluster_has_cores_and_memory(Cluster):
 
     with pytest.raises(ValueError, match=cls_name + r"cores=4, memory='\d+GB'"):
         Cluster(cores=4)
+
+
+def test_default_workers():
```

Comment:
We could modify this further to cover all clusters:

```python
@pytest.mark.parametrize(
    "Cluster",
    [PBSCluster, MoabCluster, SLURMCluster, SGECluster, LSFCluster, OARCluster],
)
def test_default_workers(Cluster):
    """For 6 cores, default is 1 workers with 3 procs, 2 threads/proc"""
    with Cluster(cores=6, memory="4GB") as cluster:
        assert " --nprocs 3" in cluster._command_template
        assert " --nthreads 2" in cluster._command_template
```

But let's only do this if there's agreement that this will be merged before the big rewrite.